src/app_model/expressions/_expressions.py

```diff
@@ -119,6 +95,29 @@ def safe_eval(expr: Union[str, bool, Expr], context: Optional[Mapping] = None) -
 class Expr(ast.AST, Generic[T]):
     """Base Expression class providing dunder and convenience methods.
 
+    This is a subclass of `ast.AST` that provides rich dunder methods that
+    facilitate joining and comparing typed expressions.  It only implements a
+    subset of ast Expressions (for safety of evaluation), but provides more
+    than `ast.literal_eval`.
+
+    Expressions that are supported:
+    - Names: 'myvar' (these must be evaluated along with some context)
+    - Constants: '1'
+    - Comparisons: 'myvar > 1' (includes `in`, `not in`)
+    - Boolean Operators: 'myvar and yourvar'
```

Comment:
We probably need an addition comment here.  I think it's actually `&` rather than `and` here... (ie we overload the bitwise and operator to mean the Boolean and)

Reply:
Done.

I noticed that `(() and (1,2))` should work for boolean `and` but gives an error here:
```python
a = parse_expression('a & b')
a.eval({'a':(), 'b': (1,2)})
```
```
TypeError: unsupported operand type(s) for &: 'tuple' and 'tuple'
```